tests/manager/manager-ctor-004.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Deprecated boolean options in URI string
```

Comment:
We previously did not have any tests for this, but now that we're testing for invalid boolean types (e.g. `"?journal=invalid"`) I think it's helpful to have some record of strings accepted by libmongoc (even if deprecated).